vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -28,46 +28,49 @@ The secondary tenant has the same capabilities as the primary tenant but with it
 |:--|:--
 | YOUR-APP-NAME | The name of the application, e.g., quickstart
 | YOUR-HOST-NAME | The base host name of the application, e.g., your-domain.com
-| YOUR-TENANT-ID | Refers to the identifying name of the current tenant and also works as a subdomain name
-| YOUR-NAMESPACE | The Kubernetes namespace name in which your app is running
-
+| YOUR-TENANT-ID | Refers to the identifying name of the current tenant and also works as a subdomain name, e.g., yoursite results in yoursite.your-domain.com for the base url of the tenant
+| YOUR-NAMESPACE | The Kubernetes namespace in which your app is running
 
-### Keycloak Configuration
-Import the realm for Keycloak from the primary tenant and reconfigure it for the secondary tenant.
-1. Follow the first step of this tutorial [Create a Backup of the Keycloak Realm](../devops/backing-restoring-keycloak.md) 
-2. Edit the generated JSON file of the realm with these updates:
-     * Remove every `id` attribute in the file.  
-     * Replace the properties `realm` & `displayName` with YOUR-TENANT-ID.
-     * Add `YOUR-TENANT-ID` + "." as a prefix to the following fields:
-`redirectUris` and `webOrigins` under `clientId` with `entando-web`, `YOUR-APP-NAME` and `YOUR-APP-NAME-de`.  
+### Keycloak
+Each tenant requires its own Keycloak realm. The following steps show how to create a tenant-specific realm in the standard Entando-deployed Keycloak instance.
 
-For example, if `YOUR-APP-NAME` is quickstart, and `YOUR-TENANT-ID` is 2ndtenant1:
+1. [Create a Backup of the Keycloak Realm](../devops/backing-restoring-keycloak.md) 
+2. Remove the `id` attributes so Keycloak will recognize the data as new entries upon import:
+``` bash
+sed -i '/"id" : "/ d' keycloak-entando-realm.json
```

Comment:
Yes it works as I shared on mac. It is because of a minor different behavior from that version to the others. According to Stackoverflow

> On the OSX version of sed, the -i option expects an extension argument so your command is actually parsed as the extension argument and the file path is interpreted as the command code.

Reply:
@avdev4j perfect, thanks. I updated the code to use that format now.